### PR TITLE
Tlportal 202

### PR DESCRIPTION
**Pull request by dlhaines**

### CODE REVIEW

File: server/WAPI.rb [hunk 1]
```diff
@@ -142,19 +149,10 @@ def get_request(request)
...
-      response = RestClient.post @token_server,
-                                 "grant_type=client_credentials&scope=PRODUCTION",
-                                 {
-                                     :Authorization => @renewal,
-                                     :content_type => "application/x-www-form-urlencoded"
-                                 }
-      ## If it worked then parse the result.  This is here to capture any JSON parsing exceptions.
+      response = runTokenRenewalPost
+      ## If it worked then parse the result as json.  This is here to capture any JSON parsing exceptions.
       if response.code == 200
```
Comment (zqian) [inline server/WAPI.rb]: should this 200 also be replace with "SUCCESS"?

File: server/WAPI.rb [hunk 2]
```diff
@@ -162,31 +160,44 @@ def renew_token
...
+    ## got no response so say that.
     if response.nil?
       logger.warn("WAPI: #{__LINE__}: error renewing token: nil response ")
-      wr = WAPIResultWrapper.new(666, "error renewing token: nil response", response)
-    elsif response.code != 200
+      return WAPIResultWrapper.new(UNKNOWN_ERROR, "error renewing token: nil response", response)
+    end
+
+    # if got an error so say that.
+    if response.code != 200
```
Comment (zqian) [inline server/WAPI.rb]: 200 => SUCCESS

File: server/WAPI.rb [hunk 3]
```diff
@@ -162,31 +160,44 @@ def renew_token
...
+      return WAPIResultWrapper.new(UNKNOWN_ERROR, "error renewing token: response code", response)
     end
-    renew.stop
-    logger.info("WAPI: renewed token: stopwatch: "+renew.pretty_summary)
-    wr
+
+    # all ok
+    print_token = sprintf "%5s", @token
+    logger.debug("WAPI: #{__LINE__}: renewed token: #{print_token}")
+    return WAPIResultWrapper.new(200, "token renewed", response)
```
Comment (zqian) [inline server/WAPI.rb]: 200 => SUCCESS

File: server/spec/test_ldap_check.rb [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
 require_relative '../ldap_check'
 
-class LdapTest < MiniTest::Test
+class TestLdap < MiniTest::Test
```
Comment (zqian) [inline server/spec/test_ldap_check.rb]: Why change the class name here?

Comment (dlhaines) [inline server/spec/test_ldap_check.rb]: Shouldn't change it.  This is from another jira.

**Pull request merged**